src/react/boosts/BoostCard.tsx

```diff
@@ -37,6 +37,13 @@ export class BoostCard extends React.Component<BoostCardProps> {
     const timeEnded = timeRemaining === "Boost Ended";
     const goalReached = this.props.boostData.paymentsTotal >= this.props.boostData.goalAmount;
     const newsroomAddress = this.props.boostData.channelID;
+    let btnText = "Support";
+    if (timeEnded) {
+      btnText = "Boost Ended";
+    } else if (!this.props.newsroomData.whitelisted) {
+      btnText = "Cannot Support";
```

Comment:
`Cannot Support` sounds awkward what about `Boost Disabled`?

Reply:
I think also leaving the button text as-is but just disabling it might be enough, the rejection notice is right below it.